entrypoint.sh

```diff
@@ -1,6 +1,9 @@
 #!/bin/bash
 set -eu
 
+git config --global safe.directory "${GITHUB_WORKSPACE}"
+git config --global safe.directory /github/workspace
```

Comment:
Why do you add both `${GITHUB_WORKSPACE}` and `/github/workspace`. I think based on https://github.com/actions/checkout/issues/766 the first line should be enough.